srml/support/src/event.rs

```diff
@@ -319,23 +318,7 @@ macro_rules! impl_outer_event {
 			$runtime;
 			$system;
 			Modules { $( $rest $(<$t>)*, )* };
-			$module::Event<$runtime>,;
-		);
-	};
-	(
-		$(#[$attr:meta])*
-		pub enum $name:ident for $runtime:ident where system = $system:ident {
-			$module:ident,
-			$( $rest:tt $( <$t:ident> )*, )*
-		}
-	) => {
-		$crate::impl_outer_event!(
-			$( #[$attr] )*;
-			$name;
-			$runtime;
-			$system;
-			Modules { $( $rest $(<$t>)*, )* };
-			$module::Event,;
+			;
```

Comment:
Are you sure that removing this is correct? :D 
The error sounds more like we need some kind of default/empty case.

Reply:
Ok maybe I should test on a full working example because my modification of node-runtime doesn't compile then for other reason.

For me what it was trying to do was starting the processing ahead of time. (and ensuring a module exist).

Also I tried to be careful with commas but it is always very hard not to make mistake.

Reply:
So you will look into this again?

Reply:
yes ok, tomorrow

Reply:
Okay :)